src/modules/admin/components/services/NewServiceTypeButton.tsx

```diff
@@ -0,0 +1,108 @@
+import { useMutation } from '@apollo/client';
+import AddIcon from '@mui/icons-material/Add';
+import { Button, DialogActions, DialogContent } from '@mui/material';
+import DialogTitle from '@mui/material/DialogTitle';
+import { Stack } from '@mui/system';
+import * as React from 'react';
+import { useState } from 'react';
+import CommonDialog from '@/components/elements/CommonDialog';
+import LabeledCheckbox from '@/components/elements/input/LabeledCheckbox';
+import TextInput from '@/components/elements/input/TextInput';
+import FormActions from '@/modules/form/components/FormActions';
+import FormSelect from '@/modules/form/components/FormSelect';
+import { isPickListOption } from '@/modules/form/types';
+import {
+  CreateServiceTypeDocument,
+  CreateServiceTypeMutation,
+  CreateServiceTypeMutationVariables,
+  PickListOption,
+  PickListType,
+  useGetPickListQuery,
+} from '@/types/gqlTypes';
+import { evictQuery } from '@/utils/cacheUtil';
+
+const NewServiceTypeButton = () => {
+  const [dialogOpen, setDialogOpen] = useState(false);
+  const [name, setName] = useState('');
+  const [supportsBulkAssignment, setSupportsBulkAssignment] = useState(false);
+  const [serviceCategory, setServiceCategory] = useState<PickListOption | null>(
+    null
+  );
+
+  const [createServiceType, { error, loading }] = useMutation<
```

Comment:
You should be able to use `useCreateServiceTypeMutation` here. It is generated into `gqlTypes.ts`